src/pool.rs

```diff
@@ -245,6 +245,27 @@ impl ConnectionPool {
                     let mut replica_number = 0;
 
                     for (address_index, server) in shard.servers.iter().enumerate() {
+                        let mut mirror_addresses: Vec<Address> = vec![];
+                        let mirror_idx = 20_000;
```

Comment:
Curious what is this magic number?

Reply:
`address_index` is used as an index in `self.databases[shard_id]` array. It is irrelevant for mirrors since we don't access mirrors outside the mirror task. So the value should not matter.

`address_id` is important. We use as an identifier for stats. So collisions should be avoided. 
For this, I split the `usize` number space into two. The second half is left for mirrors.

Reply:
I just treated mirrored addresses like normal addresses, we just keep incrementing `address_id` for each connection, mirrored or not.